docs/content/installation/install_hostdefinition.md

```diff
@@ -0,0 +1,36 @@
+# Installing the `HostDefinition` custom resource
+
+Install the `HostDefinition` custom resource to enable automatic host definitions from within the CSI driver.
+
+Use the following steps to install the `HostDefinition` custom resource, with [GitHub](https://github.com/IBM/ibm-block-csi-operator).
+
+**Note:** Only hosts added after installation are automatically defined with the CSI driver.
+
+1.  Install the operator.
+
+    1. Download the manifest from GitHub.
+
+        ```
+        curl https://raw.githubusercontent.com/IBM/ibm-block-csi-operator/v1.10.0/deploy/installer/generated/host-definition.yaml > host-definition.yaml
```

Comment:
@matancarmeli7 could you point me to the PR where this path exists?

Reply:
He said it would be in the same place as the installer so I put it in like that for now. i also assumed the file name

Reply:
updated